.github/workflows/reusable-smoke-test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - windows-latest
+          - windows-2019
```

Comment:
I totally forgot that we don't run windows tests for pull requests. If we want to run these tests on windows latest we probably have to change the base image of windows servers.